### ISSUE TITLE
Changes to LinkedIn tutorial 

### ISSUE THREAD

**Issue by bootrino** [requests/requests-oauthlib#332]
Looks like LinkedIn is making some changes that probably need to result in changes to the LinkedIn tutorial:

https://developer.linkedin.com/blog/posts/2018/redirecting-oauth-uas

**Comment by jtroussard** [comment]
Additionally (from the link updates) looks like LinkedIn is expecting `client_id` in the request body when getting the authorization verifier code from the callback url. Not sure why or where it's being left behind but setting the `fetch` method argument `include_client_id=True` corrects this issue.

ending up with something like this
```python
os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'

from requests_oauthlib import OAuth2Session
from requests_oauthlib.compliance_fixes import linkedin_compliance_fix

# Credentials you get from registering a new application
client_id = '<the id you get from linkedin>'
client_secret = '<the secret you get from linkedin>'
redirect_url = 'http://127.0.0.1'

# OAuth endpoints given in the LinkedIn API documentation (you can check for the latest updates)
authorization_base_url = 'https://www.linkedin.com/oauth/v2/authorization'
token_url = 'https://www.linkedin.com/oauth/v2/accessToken'

# Authorized Redirect URL (from LinkedIn configuration)
linkedin = OAuth2Session(client_id, redirect_uri=redirect_url)
linkedin = linkedin_compliance_fix(linkedin)
	
# Redirect user to LinkedIn for authorization
authorization_url, state = linkedin.authorization_url(authorization_base_url)
print('Please go here and authorize,', authorization_url)

# Get the authorization verifier code from the callback url
redirect_response = input('Paste the full redirect URL here:')
	
# Fetch the access token
linkedin.fetch_token(token_url,client_secret=client_secret,
include_client_id=True,authorization_response=redirect_response)
	
# Fetch a protected resource, i.e. user profile
r = linkedin.get('https://api.linkedin.com/v1/people/~')
print(r.content)```

**Comment by jtroussard** [comment]
This has been solved with PR #397
